src/test/java/buchen/station/FindClosestStationTest.java

```diff
@@ -10,61 +12,83 @@ public class FindClosestStationTest {
 
     @Test
     void closestStationWithAvailableBikes() {
-        //Given
-        CitiBikeServiceFactory factory = new CitiBikeServiceFactory();
-        CitiBikeService service = factory.getService();
-        FindClosestStation stationFinder = new FindClosestStation();
-
-        Map<String, Station> stationsMap = stationFinder.merge(service);
-        double lon = -73.971212141;
-        double lat = 40.744220;
-
-        // when
-        Station closestStation = stationFinder.closestStationAvailableBikes(stationsMap, lat, lon);
-
-        // then
-        boolean hasAvailable = closestStation.num_bikes_available > 0 || closestStation.num_ebikes_available > 0;
-        assertNotNull(closestStation);
-        assertEquals("FDR Drive & E 35 St", closestStation.name, "Expected station name to be 'FDR Drive & E 35 St'");
-        assertTrue(hasAvailable);
+        try {
```

Comment:
What I meant was wrap the calls to the S3 client in a try/catch, not the whole test